src/practitest_firecracker/practitest.clj

```diff
@@ -16,6 +16,8 @@
 ;; utils
 
 (def backoff-timeout "Backoff timeout in seconds" 20)
+(def num-attemps "Number of attempts to run" 10)
```

Comment:
typo: `num-attempts` (missing `t` before `s`)
Actually better call it `max-attempts`